functions/two_sum.py

```diff
@@ -1,2 +1,7 @@
-def twoSum(n, t):
-    pass
+def two_sum(n, t):
+    for i in range(len(n)):
+        for j in range(i + 1, len(n)):
+            if n[j] == t - n[i]:
+                return [i, j]
+
+
```

Comment:
would love to see what tests you made!

Reply:
Yes, I did forget those unfortunately. I will not be making a new code review for the updates though. Time constraints...